temporal-normalizer/tools/TimeML-Normalizer/README.TXT

```diff
@@ -0,0 +1,41 @@
+====
```

Comment:
You can keep the python evaluation. But ideally we should get rid of jar files and binary files, and anything not being used.